src/pages/events/rule-events.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Create rule events
+description: Create custom rule events using declarative configuration
+---
+
+# Create rule events
+
+You may decide that you want Adobe I/O Events for Adobe Commerce to notify the client application when certain conditions occur. For example, by default, if you register an event that tracks the remaining quantity of the product, the eventing service sends that information to your client application each time Commerce emits the event. However, you might be interested in the remaining quantity only when it reaches a specific threshold, such as 20 units. Rule events allow you to define exactly when to send events to your application. Otherwise, the client application must include code to filter out the unimportant and unnecessary data.
+
+A rule event acts as an extension of a custom or native Commerce event. You must specify the source, or parent, event and define one or more rules that evaluate the data that is present in the payload in the parent event. If all the individual rules defined in a rule event evaluate as true, then the eventing service sends the rule event to the application. If one or more rules evaluate as false, the service sends the parent event instead.
```

Comment:
For the last sentence - the parent event only would be sent if there was another subscription for the parent event separate from the rule based subscription. If one or more rules evaluate as false, nothing will be sent due to the rule event subscription.